backend/service/src/universal_codes/mod.rs

```diff
@@ -129,15 +136,33 @@ impl UniversalCodesService {
         }
     }
 
+    pub async fn pending_change(
+        &self,
+        request_id: String,
+    ) -> Result<Option<PendingChange>, UniversalCodesServiceError> {
+        let result = dgraph::pending_change(&self.client, request_id)
+            .await
+            .map_err(|e| UniversalCodesServiceError::InternalError(e.message().to_string()))?; // TODO: Improve error handling?
+
+        match result {
+            Some(pending_change) => Ok(Some(pending_change)),
+            None => Ok(None),
+        }
+    }
+
     pub async fn pending_changes(
         &self,
-        // filter: PendingChangesFilter, // TODO: need this once filtering by state?
+        // filter: PendingChangesFilter, // TODO: probably should expose this to allow filtering by state?
```

Comment:
With this comment are you thinking something like see rejected changes, or see recently approved?
I don't think it's important at this stage, although I did think of the scenario where someone accidentally approves or accidentally rejects... I think that's another example of where we need the option to edit existing codes...

Reply:
Yeah that was my thinking, if we ever are needing to view recent changes - future thing for sure!